tool/src/org/antlr/v4/codegen/target/GoTarget.java

```diff
@@ -54,6 +57,26 @@ protected void addBadWords() {
 	}
 
 	@Override
+	protected void genFile(Grammar g, ST outputFileST, String fileName) {
```

Comment:
I can't speak to whether this works as intended; I'm not familiar with the Java classes. I'd like this to be reviewed by someone else. Maybe @pboyer?